server/src/main/java/org/candlepin/model/Content.java

```diff
@@ -122,9 +137,18 @@
     @Size(max = 255)
     private String arches;
 
+    @XmlTransient
+    @Column(name = "entity_version")
+    private Integer entityVersion;
+
+    @XmlTransient
+    @Column
+    private Integer locked;
```

Comment:
SQL issues. When creating it as a boolean, Oracle complains. Letting liquibase use the pseudo-type for boolean created issues with the migration queries and typing. Dropping it to an integer was the only way I saw to make everything happy without adding a bunch of backend-specific branches everywhere.